test/libsolidity/SemanticTest.cpp

```diff
@@ -119,7 +141,18 @@ TestCase::TestResult SemanticTest::run(ostream& _stream, string const& _linePref
 	return result;
 }
 
-TestCase::TestResult SemanticTest::runTest(ostream& _stream, string const& _linePrefix, bool _formatted, bool _compileViaYul, bool _compileToEwasm)
+void SemanticTest::addBuiltin(string _module, string _function, std::shared_ptr<Builtin> _builtin)
+{
+	m_builtins[_module][_function] = _builtin;
```

Comment:
Do we really need this hierarchical separation? Can't we just allow `.` as part of the name and use a single string?

Reply:
Hmm, I guess its possible to remove this hierarchical separation.